site/en/docs/privacy-sandbox/feedback/index.md

```diff
@@ -1024,6 +1024,33 @@ features resulting in more Intent messages._**
   </tr>
 </table>
 
+#### Bounce tracking mitigations
+
+_Reduce or eliminate the ability of bounce tracking to recognize people across contexts. [Learn more about bounce tracking mitigations](https://developer.chrome.com/docs/privacy-sandbox/bounce-tracking-mitigations/)._
+
+<table class="width-full">
+  <tr>
+    <td><strong>Feedback</strong></td>
+    <td><a href="[https://github.com/bslassey/ip-blindness/issues](https://github.com/privacycg/nav-tracking-mitigations/issues)">privacycg/nav-tracking-mitigations</a></td>
```

Comment:
Shouldn't this just read <a href="https://github.com/privacycg/nav-tracking-mitigations/issues">privacycg/nav-tracking-mitigations</a>?